packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div class="kong-ui-entities-plugin-form">
+    <EntityBaseForm
```

Comment:
Question: is this just a placeholder or are we using `EntityBaseForm` to build plugin forms?

Reply:
For now just a placeholder, I didn't mean to actually check this file in, sorry about that!